src/System.Runtime.Extensions/src/ApiCompatBaseline.netcoreapp.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Runtime.Extensions:
+MembersMustExist : Member 'System.IO.Path.TrimEndingDirectorySeparator(System.ReadOnlySpan<System.Char>)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.IO.Path.TrimEndingDirectorySeparator(System.String)' does not exist in the implementation but it does exist in the contract.
+Total Issues: 2
```

Comment:
One question on this, I added this file because build checks that all members of ref project are "implemented" in some way on src. For S.R.E, we've also a reference to runtime https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj#L260 where some methods are implemented. If I understand well build succeeds because api compat check find members on runtime reference. Where I can found all "apis" surface used by api compat for `System.Private.CoreLib`? Because in case of local clr this methods won't found.
Please let me know if I'm totally off track.
/cc @jkotas @stephentoub @Anipik

Reply:
This file should not be needed when building against coreclr with your changes. I am not sure why you had to add it.

Reply:
> This file should not be needed when building against coreclr with your changes. I am not sure why you had to add it.

Because if I build from `src/System.Runtime.Extensions` api compat complain about new method inexistence...however...since it's not working anyway(I need to build from `src/System.Runtime.Extensions/tests` https://github.com/dotnet/corefx/issues/31570#issuecomment-435677560 I'll remove this unuseful file.